### DISCUSSION TITLE
[Projects Beta] Board Views

### DISCUSSION THREAD

**Discussion by RachelSek** [community/community#15648] [Projects and Issues] [answered]
I would like to create multiple Board Views. Currently, (unless I am doing something wrong) if I change the column Headers for a specific board view the changes reflect globally across ALL board views. e.g. I want a board view by Status (not started; in progress, testing, Done), Sprint #, etc...

**Comment by egost** [comment]
This would be great! I would like to be able to create my projects from a single template (or copying a project would do it too)

**Comment by matthewferry** [comment, marked as answer]
This seems to work for me (if I'm not misunderstanding what you're asking for). 

I created a new field called `Quarters`, created a new view, set it as a board, set the columns to be `Quarters` instead of `Status`, saved the view. the previous view maintains `Status` as the header, while the new view uses `Quarters`. I just created this project today, so not sure if maybe there's something weird with old projects?

https://user-images.githubusercontent.com/895885/167749548-7a83405f-26d6-4e71-ab31-93bf7228ad17.mov


To set the columns for a board view, click on columns in the menu of the board view: 
<img width="396" alt="board view columns" src="https://user-images.githubusercontent.com/895885/167749799-e728cbf8-da18-4990-a4bb-5484ffcf1f0a.png">

**Comment by RachelSek** [comment]
@matthewferry Yes!! It now seems so simple in hindsight. Thank you!
